include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 1
+#define SOF_ABI_MINOR 2
```

Comment:
@plbossart @fredoh9 would removing the preload_count from comp_config constitute a minor version update or a major one.

Alternatively, can we retain the bytes and rename them to reserved with a minor version update?

Reply:
yes, keep the fields and do a minor version update